### PR TITLE
Documentation converted to markdown format.

### PR DESCRIPTION
**Pull request by collint8**

### CODE REVIEW

File: Tools/SpeedlineTool.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+| Action      | Function          | 
+| ----------- | ----------------- |
+| left mouse  | add or move point |
+| right mouse | erase point       | 
+| F           | Paint flood fill  |
+| E           | Erase flood fill  |
+| C           | Clear Seed points |
+
+![alt text]({{ site.github.url }}/images/{{ page.title }}GUI.png)
```
Comment (a-y-khan) [inline Tools/SpeedlineTool.md:40]: Missing image? Link looks broken.

File: Tools/IntensityCorrection.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Summary
+
+This filter attempts to make a homogeneous intensity over the data layer.
+
+### Detailed Description
+
+The intensity correction filter attempts to make the intensity over the data layer homogenous. This is very useful when dealing with MR images, as the intensity values and contrast can vary over space due to magnetic field variations. This filter will attempt to fit a spline to average intensity of the image, then to use that spline to equalize the intensity values over the space.
+
+There are two parameters in this filter, * polynomial order* and *edge sensitivity.*
```
Comment (a-y-khan) [inline Tools/IntensityCorrection.md]: Formatting needs correcting.

File: Tools/ArithmeticFilter.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Description
+
+### Summary
+
+This filter runs basic boolean operations on a pixel by pixel basis.
+
+### Detailed Description
+
+The arithmetic filter allows the user to run basic arithmetic and boolean operations on both image and mask data on a pixel by pixel basis. This filter can handle up to four inputs and can be used to perform calculate any expression provided by the user(correct syntax is RESULT = <expression>;). There are also many saved expressions that exists as separate filters or tools, the main difference is the functionality to handle more than two inputs.
```
Comment (a-y-khan) [inline Tools/ArithmeticFilter.md]: Formatting of <expression> needs to be fixed.

**Review by a-y-khan — CHANGES_REQUESTED** [review]
Please go through the text and format the references to GUI elements consistently. Use a combination of bold & italic. Mouse or key descriptions are fine in bold.

**Comment by collint8** [comment]
Formatting is fixed, and italicize/bold has been standardized. The cases you mentioned have also been repaired.

**Comment by a-y-khan** [comment]
This looks really good!

File: Tools/SmoothBinaryErode.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Though there appears to be several features similar to the fast binary dilate -> erode filter, these have been disabled due to complexity issues from this filter.
+
+It should be noted that the edge of the volume provides complications to this filter. If the mask layer is touching the edge of the volume and erode is performed, the edge slice will perform a 2D erosion only, so that the mask still connects to the edge. This can become a problem when the original mask data is near the edge, but not touching, then dilate -> erode is run so that the dilate causes the mask to touch the edge. The erosion will then leave the mask touching the edge of the volume.
+
+This filter and fast binary dilate -> erode are similar in function, but are slightly different in the results. As the name suggests, smooth binary dilate -> erode will produce a smoother surface after each step due to an additional correction in the algorithm. This additional step adds to runtime dramatically. You should use this algorithm if the radii used are large, or if the erosion and dilation are performed multiple times each.
```
Comment (a-y-khan) [inline Tools/SmoothBinaryErode.md:28]: Could you please capitalize (and maybe bold or emphasize) filter names in text.

**Pull request merged**